assets/src/onboarding-wizard/pages/template-mode/get-selection-details.js

```diff
@@ -139,11 +139,11 @@ export function getSelectionText( ...args ) {
 
 		case match( COMPATIBILITY, TRANSITIONAL, MOST_RECOMMENDED, NON_TECHNICAL ):
 		case match( COMPATIBILITY, TRANSITIONAL, MOST_RECOMMENDED, TECHNICAL ):
-			return __( 'Transitional mode is recommended because it will make it easy to keep AMP content even if you install non-AMP-compatible plugins later', 'amp' );
+			return __( 'Transitional mode is recommended because it makes it easy to keep your content as valid AMP even if non-AMP-compatible plugins are installed later', 'amp' );
```

Comment:
```suggestion
			return __( 'Transitional mode is recommended because it makes it easy to keep your content as valid AMP even if non-AMP-compatible plugins are installed later.', 'amp' );
```